pulp_ansible/app/tasks/collections.py

```diff
@@ -375,33 +375,38 @@ def rebuild_repository_collection_versions_metadata(
             qkwargs["version"] = version
         qs = CollectionVersion.objects.filter(**qkwargs)
 
-    for cv in repov.get_content(content_qs=qs):
-        try:
-            rebuild_collection_version_meta(str(cv.pulp_id))
-        except Exception as e:
-            log.exception(e)
-
-
-def rebuild_collection_version_meta(collection_version_pk):
+    qs = repov.get_content(content_qs=qs)
+    with ProgressReport(
+        message=_("Rebuild collection version metadata (total)"),
+        code="rebuild_metadata.total",
+        total=qs.count(),
+    ) as ptotal, ProgressReport(
+        message=_("Rebuild collection version metadata (failed)"), code="rebuild_metadata.failed"
+    ) as pfailed:
+        for cv in qs:
+            try:
+                _rebuild_collection_version_meta(cv)
+            except Exception as e:
+                pfailed.increment()
+                log.exception(e)
+            ptotal.increment()
+
+
+def _rebuild_collection_version_meta(collection_version):
```

Comment:
Need to fix the unit tests that were added now that this function has been renamed.